migration-toolkit-for-containers/installing-mtc-restricted.adoc

```diff
@@ -0,0 +1,34 @@
+[id="installing-mtc-restricted"]
+= Installing the Migration Toolkit for Containers in a restricted network environment
+include::modules/common-attributes.adoc[]
+:context: installing-mtc-restricted
+:installing-mtc-restricted:
+
+toc::[]
+
+You can install the {mtc-full} ({mtc-short}) on {product-title} 4 in a restricted network environment.
+
+[IMPORTANT]
+====
+You must install the same {mtc-short} version on all clusters.
+====
+
+By default, the {mtc-short} web console and the `Migration Controller` pod run on the target cluster.
+
+You can configure the `Migration Controller` custom resource manifest to run the {mtc-short} web console and the `Migration Controller` pod on a link:https://access.redhat.com/articles/5064151[remote cluster].
+
+include::modules/migration-installing-mtc-on-ocp-4.adoc[leveloffset=+1]
+
+[id="configuring-replication-repository_{context}"]
+== Configuring a replication repository
+
+You must configure an object storage to use as a replication repository. The {mtc-full} ({mtc-short}) copies data from the source cluster to the replication repository, and then from the replication repository to the target cluster. Multi-Cloud Object Gateway (MCG) is the only supported option for a restricted network environment.
```

Comment:
```suggestion
You must configure object storage to use as a replication repository. The {mtc-full} ({mtc-short}) copies data from the source cluster to the replication repository, and then from the replication repository to the target cluster. Multi-Cloud Object Gateway (MCG) is the only supported option for a restricted network environment.
```
Or "You must configure an object storage [noun] to use ..." but I'm not sure what the noun would be,